src/ConsoleApplication.php

```diff
@@ -93,41 +92,20 @@ private function handleCommand(string $commandName): void
         }
 
         $handler = $consoleCommand->handler;
-
-        $params = $this->resolveParameters($handler);
+        $input = $this->argumentBag->resolveInput($consoleCommand);
```

Comment:
I think the more logical approach is that argument bag isn't responsible for resolving input  for a console command (made a comment about it in the argument bag class as well). But maybe I just need some more context for it to make sense